README.md

```diff
@@ -32,18 +32,54 @@ Cordova Android is an Android application library that allows for Cordova-based
 
 ## Requirements
 
-- Java Development Kit (JDK) 11
-- [Android SDK](https://developer.android.com/)
+* Java Development Kit (JDK) 11
+* [Android SDK](https://developer.android.com/)
 
 ## Cordova Android Developer Tools
 
 Use the [Cordova command-line tool](https://www.npmjs.com/package/cordova) to create projects and install plugins.
 
+However, the following scripts can be used instead:
+
+    ./bin/create [path package activity] ... creates the ./example app or a cordova android project
+    ./bin/check_reqs ....................... checks that your environment is set up for cordova-android development
+    ./bin/update [path] .................... updates an existing cordova-android project to the version of the framework
+
+These commands live in a generated Cordova Android project. Any interactions with the emulator require you to have an AVD defined.
+
+    ./cordova/clean ........................ cleans the project
+    ./cordova/build ........................ calls `clean` then compiles the project
+    ./cordova/log   ........................ streams device or emulator logs to STDOUT
+    ./cordova/run   ........................ calls `build` then deploys to a connected Android device. If no Android device is detected, will launch an emulator and deploy to it.
+    ./cordova/version ...................... returns the cordova-android version of the current project
+
+To use a **shared CordovaLib**, for example in development, link the appropriate cordova-android platform folder path:
+
+```bash
+cordova platform add --link /path/to/cordova-android
+```
+
 ## Using Android Studio
 
 1. Create a project
 2. Import it via "Non-Android Studio Project"
 
-## Running the Native Tests
+## Updating a Cordova project
+
+When you install a new `cordova-cli` version that comes with a new Android platform version, from within your project:
+
+```bash
+cordova platform rm android
+cordova platform add android
+```
+
+## How to Test Repo Development
+
+```bash
+npm install
+npm test
+```
+
+## Futher reading
 
-The `test/` directory in this project contains an Android test project that can be used to run different kinds of native tests. Check out the [README contained therein](test/README.md) for more details!
+* [https://cordova.apache.org/](https://cordova.apache.org/)
```

Comment:
```suggestion
* [Apache Cordova](https://cordova.apache.org/)
```

Reply:
Changes on Readme moved to PR #1308 ; and I commited this suggestion